oidcclient/jwk.go

```diff
@@ -21,19 +21,27 @@ import (
 )
 
 const defaultJwkExpiration = 15 * time.Minute
-const zoneIDHeader = "x-zone_uuid"
+const appTIDHeader = "x-app_tid"
+const clientIDHeader = "x-client_id"
+const azpHeader = "x-azp"
 
-// OIDCTenant represents one IAS tenant correlating with one zone with it's OIDC discovery results and cached JWKs
+// OIDCTenant represents one IAS tenant correlating with one app_tid and client_id with it's OIDC discovery results and cached JWKs
 type OIDCTenant struct {
 	ProviderJSON    ProviderJSON
-	acceptedZoneIds map[string]bool
+	acceptedTenants map[TenantInfo]bool
 	httpClient      *http.Client
 	// A set of cached keys and their expiry.
 	jwks       jwk.Set
 	jwksExpiry time.Time
 	mu         sync.RWMutex
 }
 
+type TenantInfo struct {
```

Comment:
TenantInfo sounds somewhat misleading as the azp can be any other authorized client (not 100% sure, but i think it can also be another tenant) it doesn't described just a tenant. I think it's rather an OIDC client info or relying party or something in that direction.

Reply:
Good point.
How about "ClientInfo" since we are already in the jwk package?